lib/jsonapi/view.ex

```diff
@@ -56,6 +56,23 @@ defmodule JSONAPI.View do
         def relationships, do: []
       end
 
+  Fields may be omitted manually using the `hidden/1` function.
+
+      defmodule UserView do
+        use JSONAPI.View
+
+        def fields, do: [:id, :username, :email]
+
+        def type, do: "user"
+
+        def hidden(_data) do
+          [:email] # will be removed from the response
+        end
```

Comment:
One thing that has stumped me so far is the need for this as a public API.  Since it seems one would just modify `fields` to not include `email`.  Is there a reason to have this as a public API you think?

(btw I am so sorry for going back and forth.  I should have gotten these comments all rolled up into one review :()

Reply:
`hidden/1` comes from #126. The gist seems to be conditionally excluding fields given the data available. I think an example scenario is hiding a sensitive field from users with lower privilege.

Anyways, given that it's been part of the public API for a while we ought to leave it as such.

Reply:
Oh you are right.

In the past, for lower privileges, I take advantage of checking the logic in a function for that specific field.  But perhaps this allows bulk sending in fields to hide.

Reply:
Documenting as such still seems weird but since it has been there 👍